rpc_jobs/rpc_aio.yml

```diff
@@ -70,7 +84,20 @@
 
 - job-template:
     # DEFAULTS
-    STAGES: "Allocate Resources, Connect Slave, Prepare Deployment, Deploy RPC w/ Script, Setup MaaS, Verify MaaS, Install Tempest, Tempest Tests, Cleanup, Destroy Slave"
+    STAGES: >-
+      Allocate Resources,
+      Connect Slave,
+      Prepare Deployment,
+      Deploy RPC w/ Script,
+      Setup MaaS,
+      Verify MaaS,
+      Install Tempest,
+      Tempest Tests,
+      Prepare Kibana Selenium,
+      Kibana Tests,
```

Comment:
Hmm, I'm not sure why it took so much longer as I can't see the breakdown by stage in the UI.  The onmetal multi-nodes seem to spend only a few mins completing the kibana tests, so I'll run another job to make sure we don't have a massive time regression here.